.github/dependabot.yml

```diff
@@ -4,23 +4,25 @@ updates:
   - package-ecosystem: "github-actions"
     directory: "/"
```

Comment:
Wonder why this is "/" instead of "/.github" 🤔 

Its working now and is only considering Github Actions but just curious (nothing blocking this PR)